demo/agent_monitor/tenant_agent_monitor.py

```diff
@@ -33,7 +33,7 @@
 
 logger = keylime_logging.init_logging("agent_monitor")
 
-initscript = None
+init_script = None  # pylint: disable-msg=C0103
```

Comment:
I think this would be more user-friend in a file: disable=invalid-name

The rest looks good. Thanks for doing this.

Reply:
I'll keep that in mind but this particular line is no longer there after I addressed your other comment.